api/CharacterData.json

```diff
@@ -48,6 +48,55 @@
           "deprecated": false
         }
       },
+      "after": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/CharacterData/after",
+          "spec_url": "https://dom.spec.whatwg.org/#ref-for-dom-childnode-after①",
+          "support": {
+            "chrome": {
+              "version_added": "54"
+            },
+            "chrome_android": {
+              "version_added": "54"
+            },
+            "edge": {
+              "version_added": "17"
+            },
+            "firefox": {
+              "version_added": "49"
+            },
+            "firefox_android": {
+              "version_added": "49"
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": "39"
+            },
+            "opera_android": {
+              "version_added": "41"
+            },
+            "safari": {
+              "version_added": "10"
+            },
+            "safari_ios": {
+              "version_added": "10"
+            },
+            "samsunginternet_android": {
+              "version_added": "6.0"
+            },
+            "webview_android": {
+              "version_added": "54"
+            }
+          },
+          "status": {
+            "experimental": true,
```

Comment:
I don't think any of these count as experimental at this point :)

Reply:
Oh yes, I fixed this in content, too.